src/libraries/Native/Windows/CMakeLists.txt

```diff
@@ -1,7 +1,8 @@
 cmake_minimum_required(VERSION 3.14.5)
+cmake_policy(SET CMP0091 NEW)
```

Comment:
we're wrapping this in a cmake version check elsewhere:
https://github.com/dotnet/runtime/blob/05cd29c81434aae62d0cb680768d0229c8da0d2d/src/installer/corehost/CMakeLists.txt#L3-L5

Reply:
Our minimum version of CMake for Windows is 3.16.0 and this CMake entry-point is only used on Windows, so we shouldn't need the version check.

Reply:
Ok but then we should bump the `cmake_minimum_required` line above to 3.16.0.